### DISCUSSION TITLE
[求助]要不要抽青衣

### DISCUSSION THREAD

**Discussion by JJC-555** [share121/inter-knot#127] [其他]
![1723200752533b384f3d053841e1e7a4c358c99860da1ba888d30781a563f6e87ca39ed4b8fa7 0](https://github.com/user-attachments/assets/a358de09-09a5-41ad-a884-9d114279c10c)
青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？青衣值不值得抽啊？

**Comment by yamtru** [comment]
我是xp党，我冲了

**Comment by zhuang1142** [comment]
抽啊，为什么不抽？

**Comment by DtarDomeExploit** [comment]
首先XP大于一切，其次，对击破位有需求的可以抽

**Comment by qiu-1214** [comment]
我是会抽一下的，主要是我有朱鸢

**Reply by minghhh233** [reply]
这个建模没欲望，不改建模绝对不抽
